### PR TITLE
Update control-access-from-unmanaged-devices.md

### PR DESCRIPTION
**Pull request by SanjoyanM**
> For new tenants apps using ACS app-only token is disabled by default. It is recommended to use Azure AD app-only model which is modern and securer. But you can change the behavior by running ‘set-spotenant -DisableCustomAppAuthentication $false' (need latest SharePoint admin powershel)

We have added this above additional option for security.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md [hunk 1]
```diff
@@ -183,6 +183,8 @@ Blocking access and blocking download may impact the user experience in some app
 > Apps that run in "app-only" mode in the service, like antivirus apps and search crawlers, are exempted from the policy.
 > 
 > If you're using classic SharePoint site templates, site images may not render correctly. This is because the policy prevents the original image files from being downloaded to the browser. 
+>
+> For new tenants apps using ACS app-only token is disabled by default. It is recommended to use Azure AD app-only model which is modern and securer. But you can change the behavior by running ‘set-spotenant -DisableCustomAppAuthentication $false' (need latest SharePoint admin powershel)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md:187]:
```suggestion
> For new tenants, apps using an ACS app-only access token is disabled by default. We recommend using the Azure AD app-only model which is modern and more secure. But you can change the behavior by running ‘set-spotenant -DisableCustomAppAuthentication $false' (needs the latest SharePoint admin PowerShell).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions only.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by SanjoyanM** [comment]
Hi all,
when can I expect my changes to be published.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**